docs/kernel-policy.md

```diff
@@ -81,12 +82,49 @@ the minimal configuration for a guest kernel for a successful microVM boot is:
     - `CONFIG_VIRTIO_BLK=y`
   - x86_64
     - `CONFIG_VIRTIO_BLK=y`
-    - `CONFIG_VIRTIO_MMIO_CMDLINE_DEVICES=y`
+    - `CONFIG_ACPI=y`
+    - `CONFIG_PCI=y`
     - `CONFIG_KVM_GUEST=y`.
 
 *Optional*: To enable boot logs set `CONFIG_SERIAL_8250_CONSOLE=y` and
 `CONFIG_PRINTK=y` in the guest kernel config.
 
+##### Booting with ACPI (x86_64 only):
+
+Firecracker supports booting kernels with ACPI support. The relevant
+configurations for the guest kernel are:
+
+- `CONFIG_ACPI=y`
+- `CONFIG_PCI=y`
+
+Please note that Firecracker does not support PCI devices. The `CONFIG_PCI`
+option is needed for ACPI initialization inside the guest.
+
+ACPI supersedes the legacy way of booting a microVM, i.e. via MPTable and
+command line parameters for VirtIO devices.
+
+We suggest that users disable MPTable and passing VirtIO devices via kernel
+command line parameters. These boot mechanisms are now deprecated. Users can
+disable these features by disabling the corresponding guest kernel configuration
+parameters:
+
+- `CONFIG_X86_MPPARSE=n`
+- `CONFIG_VIRTIO_MMIO_CMDLINE_DEVICES=n`
+
+During the deprecation period Firecracker will continue to support the legacy
+way of booting a microVM. Firecracker will be able to boot kernels with the
+following configurations:
+
+- Only ACPI
+- Only legacy mechanisms
+- Both ACPI and legacy mechanisms
+
+When booting with kernels that support both ACPI and legacy boot mechanisms
```

Comment:
I think this can go in the Caveats section starting with something like : "When booting with kernels that support both .. users may see some initialisation failures which they can ignore for the following reasons:" and this follows the explanation provided.

Reply:
Moved it! Good catch.